common/src/test/java/org/apache/nemo/common/DAGTest.java

```diff
@@ -62,28 +62,28 @@ public void testSimpleDAG() {
 
     final DAG<IntegerVertex, Edge<IntegerVertex>> dag = dagBuilder.build();
 
-    assertEquals(dag.getVertices().size(), 5);
```

Comment:
In this file, each team has its own lines of code to fix. For Team16, lines 118 and 119 are assigned. Could you please revert the codes except for those?

Reply:
I will fix line 118 and 119